airbyte-integrations/connectors/source-trello/source_trello/source.py

```diff
@@ -107,7 +107,13 @@ class Cards(ChildStreamMixin, TrelloStream):
 
     parent_stream_class = Boards
     limit = 20000
-    extra_params = {"customFieldItems": "true"}
+    extra_params = {
+        "customFieldItems": "true",
+        "pluginData": "true",
+        "actions_display": "true",
+        "members": "true",
+        "list": "true",
+    }
```

Comment:
can you comment above or point to documentation on how adding these is helping or what difference it makes

Reply:
the documentation is provided in the function docs. it points to the exact API thats being used, where the documentation for these `extra_params` is detailed. The param names also are very self-explanatory